File: Cordova/www/services/people_service.js
angular.module('myApp').service('peopleService', function () {

    this.people = [
        { firstname: 'Simona', lastname: 'Morasca', img: 'photo01.jpg' },
        { firstname: 'Mitsue', lastname: 'Tollner', img: 'photo02.jpg' },
        { firstname: 'Leota', lastname: 'Dilliard', img: 'photo03.jpg' },
        { firstname: 'Sage', lastname: 'Wieser', img: 'photo04.jpg' },
        { firstname: 'Kris', lastname: 'Marrier', img: 'photo05.jpg' },
        { firstname: 'Minna', lastname: 'Amigon', img: 'photo06.jpg' },
        { firstname: 'Abel', lastname: 'Maclead', img: 'photo07.jpg' },
        { firstname: 'Kiley', lastname: 'Caldarera', img: 'photo08.jpg' },
        { firstname: 'Graciela', lastname: 'Ruta', img: 'photo09.jpg' },
        { firstname: 'Cammy', lastname: 'Albares', img: 'photo10.jpg' },
        { firstname: 'Mattie', lastname: 'Poquette', img: 'photo11.jpg' },
        { firstname: 'Meaghan', lastname: 'Garufi', img: 'photo12.jpg' }
        //{ firstname: 'Gladys', lastname: 'Rim', img: 'photo13.jpg' },
        //{ firstname: 'Yuki', lastname: 'Whobrey', img: 'photo14.jpg' },
        //{ firstname: 'Fletcher', lastname: 'Flosi', img: 'photo15.jpg' },
        //{ firstname: 'Bette', lastname: 'Nicka', img: 'photo16.jpg' }
    ];

    // Retourne le message reçu précédé de la date et de l'heure,
    // avec le niveau d'alerte voulu
    this.get_people = function () {
        return this.people;
    };

    this.get = function(index) {
        return this.people[index];
    };

});

File: XamarinForms/ContactsXamarin/ContactsXamarin/Services/PeopleService.cs
using System.Collections.Generic;

namespace ContactsXamarin.Services
{
    public static class PeopleService
    {
        private static readonly IList<Person> Persons = new List<Person>
        {
            new Person{ FirstName = "Simona", LastName = "Morasca", Image = "photo01.jpg" },
            new Person{ FirstName = "Mitsue", LastName = "Tollner", Image = "photo02.jpg" },
            new Person{ FirstName = "Leota", LastName = "Dilliard", Image = "photo03.jpg" },
            new Person{ FirstName = "Sage", LastName = "Wieser", Image = "photo04.jpg" },
            new Person{ FirstName = "Kris", LastName = "Marrier", Image = "photo05.jpg" },
            new Person{ FirstName = "Minna", LastName = "Amigon", Image = "photo06.jpg" },
            new Person{ FirstName = "Abel", LastName = "Maclead", Image = "photo07.jpg" },
            new Person{ FirstName = "Kiley", LastName = "Caldarera", Image = "photo08.jpg" },
            new Person{ FirstName = "Graciela", LastName = "Ruta", Image = "photo09.jpg" },
            new Person{ FirstName = "Cammy", LastName = "Albares", Image = "photo10.jpg" },
            new Person{ FirstName = "Mattie", LastName = "Poquette", Image = "photo11.jpg" },
            new Person{ FirstName = "Meaghan", LastName = "Garufi", Image = "photo12.jpg" }
        };

        public static IList<Person> GetAll()
        {
            return Persons;
        }

        public static Person GetByIndex(int index)
        {
            return Persons[index];
        }
    }
}
